cocos/2d/CCNode.h

```diff
@@ -142,6 +143,15 @@ class CC_DLL Node : public Ref
     /// @{
     /// @name Setters & Getters for Graphic Properties
 
+    /*
+    setZLocalOrderOfArrive totally control the render order, it use for sometime node should cover the friendnode and sometime under the friendnode
+    */
+    void setZLocalOrderOfArrive(std::int64_t nZOrderArrived);
```

Comment:
local z order of arrival is not visible outside, we take a lot of effect to make it as it is now.